hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/AtomicCryptoTransferHTSSuite.java

```diff
@@ -133,7 +137,8 @@ public List<HapiSpec> getSpecsInSuite() {
         });
     }
 
-    @HapiTest
+    // This test is failing against mono-service.
```

Comment:
In the PR description, you wrote,

> After adding the CRYPTO_TRANSFER_V2 in the isClassicCall method, the tests failing in mono also started to fail in modularized code.

Does that mean this test now fails as a `@HapiTest` too?

Reply:
Yes, exactly

Reply:
Ah got it 💯